### ISSUE TITLE
Confusion Matrix image in 2-4-Logistic

### ISSUE THREAD

**Issue by RyanXinOne** [microsoft/ML-For-Beginners#237]
This issue is raised from [PR#235](https://github.com/microsoft/ML-For-Beginners/pull/235).

It seems that an error appears in the image of *Confusion Matrix* in [2-4-Logistic](https://github.com/microsoft/ML-For-Beginners/blob/main/2-Regression/4-Logistic/README.md).

![Confusion Matrix](https://raw.githubusercontent.com/microsoft/ML-For-Beginners/main/2-Regression/4-Logistic/images/confusion-matrix.png)

The *Actual Values* should be presented by axis 0 and *Predicted Values* in axis 1. So that calculation results can match the `classification_report` with the correct definition of precision and recall.

Finally, The correct matrix should be:

|||
| :-: | :-: |
| TP | FN |
| FP | TN |

See [Confusion Matrix in sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html?highlight=confusion_matrix#sklearn.metrics.confusion_matrix) and [Wiki](https://en.wikipedia.org/wiki/Confusion_matrix).

**Comment by jlooper** [comment]
Removing this image as it's.... confusing! :) and editing the lesson
